src/DocumentFormat.OpenXml/GeneratedCode/schemas_microsoft_com_office_spreadsheetml_2011_1_ac.g.cs

```diff
@@ -13,7 +13,7 @@
 using System.Collections.Generic;
 using System.IO.Packaging;
 
-namespace DocumentFormat.OpenXml.Office2010.ExcelAc
+namespace DocumentFormat.OpenXml.Office2013.ExcelAc
```

Comment:
you cannot change this - it's set for now (that's a breaking change)

Reply:
reverted.